### PR TITLE
[PR] Fix hierarchy handling problem. #735

### PR DESCRIPTION
**Pull request by le-cds**

### CODE REVIEW

**Comment by le-cds** [comment]
The unit tests succeed on my machine.

File: plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/options/HierarchyHandling.java [hunk 1]
```diff
@@ -10,45 +10,42 @@
...
-     * Inherit the behavior from the parent node. If set on the root node, SEPARATE_CHILDREN is
-     * assumed.
+     * Inherit the parent node's hierarchy handling. The root node has no parent node; here, this setting defaults to
+     * {@link #SEPARATE_CHILDREN}.
      */
     INHERIT,
 
     /**
-     * Include the children in the parent layout run.
+     * Allows the node's children should be included in the current layout run. Which children are included in the
```
Comment (uruuru) [inline plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/options/HierarchyHandling.java:39]: The first sentence seems to be off.

**Comment by uruuru** [comment]
I re-ran the pipeline. The tests succeed now. Ignore elkjs for now, still no idea where that issue comes from.

File: plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/options/HierarchyHandling.java (hunk 1 above)
Comment (le-cds) [reply plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/options/HierarchyHandling.java]: Fixed.

**Pull request merged**